algorithms/NarcissisticNumber.js

```diff
@@ -1,10 +1,10 @@
-function narcissistic(value) {
-    let arr = Array.from(String(value), Number)
-
-    let newArr = Array.from(String(value), Number).map( i=> (i ** arr.length))
-
-    return newArr.reduce((i, j) => i + j) === value;
-}
-
-console.log(narcissistic(153))
+function narcissistic(value) {
+    let arr = Array.from(String(value), Number)
+
+    let newArr = Array.from(String(value), Number).map( i=> (i ** arr.length))
```

Comment:
You may as well substitute it with 
`return value === arr.map(i => (i ** arr.length)).reduce((i, j) => i + j);`